### DISCUSSION TITLE
Inline `ftd.text` like span

### DISCUSSION THREAD

**Discussion by gsalunke** [fastn-stack/fastn#705] [Ideas & RFCs]
Can we have inline `ftd.text` to create UI like this?

<img width="869" alt="image" src="https://user-images.githubusercontent.com/68585007/226071707-ed3d686a-1ae0-4763-8f03-a14d11d33698.png">

Ref: [JSONBIN.io](https://jsonbin.io/api-reference/bins/read#route)

With combination of `ftd.row` and `ftd.text` we are not able to create above UI, because `ftd.row` aligns all its children horizontally like this:

<img width="740" alt="image" src="https://user-images.githubusercontent.com/68585007/226072475-d7d6d304-82d7-4187-aa93-edff8bd3e0f7.png">

**Comment by amitu** [comment]
@Heulitig when you are free please pick this one up. 

```ftd
-- ftd.text: hello
display: inline
```

We will add a new property `ftd.display display: block`. The default value would be false so it does not do any backward breaking compatibility issue. 

```ftd
-- or-type ftd.display:

-- string constant inline: inline
-- string constant block: block
-- string constant inline-block: inline-block

-- end: ftd.display
```

CC: @Arpita-Jaiswal
